csharp-advanced-topics/ActionAndFuncDelegates/ActionAndFuncDelegates/ActionAndFuncDelegates.csproj

```diff
@@ -0,0 +1,8 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net5.0</TargetFramework>
+  </PropertyGroup>
```

Comment:
We're trying to use the newest versions of the framework. If we proceed further, please make sure to install .NET 6 SDK.